coopterrones/color-palette
# Feature/save palette

#### What's this PR do?
- builds out save palette functionality 
- palettes save to state 
- adds router to app 
#### Where should the reviewer start?
- `App.js`
- `SavePaletteForm.js` 

#### How should this be manually tested?
- test by adding a palette to save and submit 
- check state for saved palettes updating in App
